File: java/210525_class/src/com/cm/carApp/util/RandData.java
package com.cm.carApp.util;

import java.util.Random;

public class RandData {
	
	Random r = new Random();
	
	String name[] = {"홍길동", "김길동", "박길동", "이길동", "최길동"};
	String tel[] = {"010-1234-5678", "010-4321-5678", "010-3478-1278", "010-4523-1978",
			"010-7890-2134"};
	String address[] = {"대구시 동구 신암4동", "서울시 동구 신암 4동", "부산시 동구 신암4동",
			"인천시 동구 신암4동", "광주시 동구 신암4동"};
	String model[] = {"SM6", "쏘나타", "싼타페", "K7", "그랜져"};
	String color[] = {"블랙", "은색", "흰색", "회색", "빨강"};
	int year[] = {2016, 2017, 2016, 2017, 2016};
	String company[] = {"르노삼성", "현대", "현대", "기아", "현대"};
	
	public String getName() {
		return name[r.nextInt(5)];
	}
	public String getTel() {
		return tel[r.nextInt(5)];
	}
	public String getAddress() {
		return address[r.nextInt(5)];
	}
	public String getModel() {
		return model[r.nextInt(5)];
	}
	public String getColor() {
		return color[r.nextInt(5)];
	}
	public int getYear() {
		return year[r.nextInt(5)];
	}
	public String getCompany() {
		return company[r.nextInt(5)];
	}
	
	

}

File: C#/210610/210610_class_car/210610_class_car/utill/RandData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _210610_class_car.carApp.utill
{
    class RandData
    {
        string[] name = {"한지민", "한효주", "이병헌", "유아인", "신예은" };
        string[] tel = { "010-3484-3123", "010-7961-3549", "010-7635-1734", "010-7985-3548", "010-7635-1777" };
        string[] address = { "대구 동구 동부로 13", "부산 해운대구 해변길 24", "서울 강남구 가로수길 15", "경남 함양군 유동길 16", "대구 달서구 감삼길 17" };
        string[] model = { "R8", "랭글러", "EQ900", "파나메라 GTS", "마이바흐 S클래스" };
        string[] color = {"블랙", "레드", "블루", "화이트", "실버" };
        int[] year = {2016, 2017, 2018, 2019, 2020 };
        string[] brand = { "아우디", "벤츠", "포르쉐", "지프", "현대" };

        Random r = new Random();
        public string getName()
        {
            return name[r.Next(0, 5)];
        }

        public string getTel()
        {
            return tel[r.Next(0, 5)];
        }

        public string getAddress()
        {
            return address[r.Next(0, 5)];

        }

        public string getModel()
        {
            return model[r.Next(0, 5)];
        }

        public string getColor()
        {
            return color[r.Next(0, 5)];
        }

        public int getYear()
        {
            return year[r.Next(0, 5)];

        }

        public string getBrand()
        {
            return brand[r.Next(0, 5)];
        }
    }

   
}
